docs/tutorials/benchmarking/README.md

```diff
@@ -0,0 +1,17 @@
+# Benchmarking in Syne Tune
+
+
+This tutorial provides an overview of tooling which facilitates benchmarking of
+HPO algorithms in Syne Tune. The same tooling can also be used to rapidly
+create launcher scripts for any HPO experiment you may want to run, allowing you
+to easily switch between local, SageMaker, and simulator back-end.
+
+
+# Table of Contents
+
+* [Benchmarking with Simulator Back-end](bm_simulator.md)
+* [Benchmarking with Local Back-end](bm_local.md)
```

Comment:
not related to this one, but somewhere in tutorials we should mention python backend too. It is used in examples scripts in `/examples` folder.

Reply:
We need a dedicated place where we compare all different back-ends, does not exist right now. But the Python backend is not so useful for benchmarking, I think.